content/blog/2018-07-26-educollab-community.md

```diff
@@ -0,0 +1,54 @@
+
+In this final post, we propose a framework for developing an rOpenSci Educators' Collaborative: a community of practice for people interested and engaged in science education using R. A community of practice is defined as a group of skilled practitioners who share a common passion, and who learn how to do it better by interacting with each other regularly. To develop this kind of community, we need to go beyond shared domains of interest in R and education. Here, we’d like to invite educators to join us in discussing how to put the other ingredients in place. 
+
+First, we need a community, a place to share information and build relationships. It is important to note that while we recognize (and envy!) The Carpentries and the strong [education community of practice](https://cookbook.carpentries.org) that has been developed around their workshops for teaching foundational coding and data science skills, we believe there is a place for a community of science educators who teach *with* R, but who may not identify as someone who teaches coding or programming skills. Currently, one of the main challenges for these educators is the availability and discoverability of course materials. There is an abundance of tutorials and a lack of curricular resources. For example, there are many individual tutorials, focused workshops or bootcamp materials, online courses, and lesson plan modules. All of these are great resources, and can be integrated within a long-form course curriculum. However, while it is helpful to leverage existing materials, adapting these materials in the classroom still requires preparation time. To use them successfully, educators usually need to think carefully about the dependencies of each individual resource (i.e., packages used, assumed prior R knowledge, assumed content domain knowledge), and how it fits into the larger temporal sequence of course materials and learning objectives. It also can be difficult to teach a class or lead a lab-based on others’ materials, written in someone else’s voice, and educators typically need to adapt the data or code to better meet their class’ needs or skill level. 
+
+On the other hand, there is a lack of integrated long-form  resources, where educators share full-length syllabi along with R-based learning materials. There are many reasons why these materials may not be open: lack of instructor confidence in sharing their teaching materials, lack of knowledge about tools and platforms for public sharing, and lack of support from institutions and departments. We would like to invite educators to begin to share their curricular materials, and offer a few recommended resources:
+
+- [Humanities Commons](https://hcommons.org)
+- [Open Science Framework](https://osf.io)
+- [GitHub repository](https://github.com) with recommended topic tags `teaching` and `rstats`.
```

Comment:
I'm naive. Maybe another reader is too. Could you link to search result of GitHub with topic tags `teaching` and `rstats`? I'm not having success finding how to search for `teaching` AND `rstats` and getting 8 results

Reply:
Added a link to a sample search